src/Symfony/Component/Notifier/Bridge/Iqsms/README.md

```diff
@@ -3,6 +3,19 @@ Iqsms Notifier
 
 Provides Iqsms integration for Symfony Notifier.
 
+DSN example
+-----------
+
+```
+// .env file
+IQSMS_DSN=iqsms://LOGIN:PASSWORD@default?from=FROM
+```
+
+where:
+ - `LOGIN` is your IQSMS login
+ - `PASSWORD` is your PASSWORD password
```

Comment:
```suggestion
 - `PASSWORD` is your password
```

Reply:
This is already resolve in https://github.com/symfony/symfony/pull/39096/commits/e667e8fcc0bac0990710a3532ed8343e96d735a2